centralstore/smartwrapper.go

```diff
@@ -34,13 +36,14 @@ type SmartWrapperOptions struct {
 }
 
 // NewSmartWrapper creates a new SmartWrapper.
-func NewSmartWrapper(options SmartWrapperOptions, basic BasicStorer) *SmartWrapper {
+func NewSmartWrapper(options SmartWrapperOptions, basic BasicStorer, tracer trace.Tracer) *SmartWrapper {
```

Comment:
Would it make sense to add the tracer to the `SmartWrapperOptions` and default as `noopTracer`?

Reply:
Two things:
* SmartWrapper doesn't use the tracer, the test program does so it doesn't really belong there.
* I think of the individual SmartWrapperOptions as things you can specify in configuration text (not complex types). I expect them to move into config eventually.